Libplanet/Blockchain/BlockChain.MineBlock.cs

```diff
@@ -299,17 +299,6 @@ void WatchTip(object target, (Block<T> OldTip, Block<T> NewTip) tip)
 
                 if (storedNonces[tx.Signer] <= tx.Nonce && tx.Nonce == nextNonces[tx.Signer])
                 {
-                    if (Policy.ValidateNextBlockTx(this, tx) is { } tpve)
```

Comment:
This deletion will cause unexpected semantic changes. (it will throw a tx exception instead of ignoring). I think that modifying this predicate as below would be more acceptable than deletion.

```csharp
                    if (!(Policy.ValidateNextBlockTx(this, tx) is null))
```